packages/browser/src/Session.ts

```diff
@@ -157,8 +154,12 @@ export class Session extends EventEmitter {
       url
     );
     if (sessionInfo) {
+      if (sessionInfo.isLoggedIn) {
```

Comment:
Maybe just a comment to explain why this event is being fired here (i.e. due to the redirect)?

Reply:
Fixed in 7c074ba3c132fef316c992aa33d11445ca311e90